Frameworks/UIKit.Xaml/ScrollView.xaml.h

```diff
@@ -0,0 +1,65 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+// clang-format does not like C++/CX
+// clang-format off
+#pragma once
+
+#include "ScrollView.g.h"
+#include "Layer.xaml.h"
```

Comment:
public interface class ILayer  is defined in there. Might as well move it out.

---
In reply to: [106713163](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106713163) [](ancestors = 106713163)